nix/hm-module.nix

```diff
@@ -162,6 +162,77 @@ in {
       ];
     };
 
+    images = mkOption {
+      description = "Image configurations";
+      type = listOf (submodule {
+        options = {
+          monitor = mkOption {
+            description = "The monitor to draw an image";
+            type = str;
+            default = "";
+          };
+
+          path = mkOption {
+            description = "The path to source image";
+            type = str;
+            default = "/home/me/cutie.png"; # only png supported for now
+          };
+
+          size = mkOption {
+            description = "Size of the image. Lesser side is chosen if not 1:1 aspect ratio";
+            type = int;
+            default = 150;
+          };
+
+          rounding = mkOption {
+            description = "The rounding of the image";
+            type = int;
+            default = -1;
+          };
+
+          frame_size = mkOption {
```

Comment:
This is now `border`, I think you forgot to change it.

Reply:
oops true. done